### ISSUE TITLE
Vulkan Mobile crash on Windows 10 (Nvidia GTX 1060)

### ISSUE THREAD

**Issue by E72UJ** [godotengine/godot#96316]
### Tested versions

Godot v4.4.dev1 - Windows 10.0.22631 - Vulkan (Mobile) - dedicated GeForce GTX 1060 - Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz (12 Threads) Can reproduce correctly

### System information

Godot v4.4.dev1 - Windows 10.0.22631 - Vulkan (Mobile) - dedicated GeForce GTX 1060 - Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz (12 Threads)

### preface
I have added many Animation player nodes to my program, among which the Text of the Animation player node is dynamically loaded, meaning the resource directory is external. The program's running directory structure is as follows:

![Snipaste_2024-08-30_13-56-02](https://github.com/user-attachments/assets/edb64fe0-b7b8-40b2-ae03-1e0a8c092f7e)
### bug
![Snipaste_2024-08-30_13-49-22](https://github.com/user-attachments/assets/03d68141-d3ca-456e-bad4-0d0806eaaf72)

After loading, a corresponding error was encountered, which was only successfully reproduced in version 4.4. In version 4.3, another error was encountered.

Similarly, in the [Godot v4.3. stable - Windows 1.0.22631- Vulkan (Mobile) - dedicated GeForce GTX 1060- Intel (R) Core (TM) i7-8750H CPU @ 2.20GHz (12 Threads)] version, the error has changed to
![Snipaste_2024-08-30_14-01-31](https://github.com/user-attachments/assets/2c4e6b8d-2933-4ec9-951c-18cd43b930a3)

### error code 
4.4:
```
CrashHandlerException: Program crashed with signal 4
Engine version: Godot Engine v4.4.dev1.official (28a72fa4344befeae251d8e9b11b84acd601a244)
Dumping the backtrace. Please include this when reporting the bug to the project developer.
[1] error(-1): no debug info in PE/COFF executable
[2] error(-1): no debug info in PE/COFF executable
[3] error(-1): no debug info in PE/COFF executable
[4] error(-1): no debug info in PE/COFF executable
[5] error(-1): no debug info in PE/COFF executable
[6] error(-1): no debug info in PE/COFF executable
[7] error(-1): no debug info in PE/COFF executable
[8] error(-1): no debug info in PE/COFF executable
[9] error(-1): no debug info in PE/COFF executable
[10] error(-1): no debug info in PE/COFF executable
[11] error(-1): no debug info in PE/COFF executable
[12] error(-1): no debug info in PE/COFF executable
[13] error(-1): no debug info in PE/COFF executable
[14] error(-1): no debug info in PE/COFF executable
[15] error(-1): no debug info in PE/COFF executable
[16] error(-1): no debug info in PE/COFF executable
[17] error(-1): no debug info in PE/COFF executable
-- END OF BACKTRACE --
```
4.3
```
ERROR: Condition "err != VK_SUCCESS" is true. Returning: FAILED
   at: command_queue_execute_and_present (drivers/vulkan/rendering_device_driver_vulkan.cpp:2266)
ERROR: Condition "err != VK_SUCCESS" is true. Returning: FAILED
   at: fence_wait (drivers/vulkan/rendering_device_driver_vulkan.cpp:2066)
ERROR: Condition "err != VK_SUCCESS" is true. Returning: FAILED
   at: command_queue_execute_and_present (drivers/vulkan/rendering_device_driver_vulkan.cpp:2266)
ERROR: Condition "err != VK_SUCCESS" is true. Returning: FAILED
```

### Steps to reproduce

This resource is very huge, mainly consisting of. png files that exceed the size limit of GitHub. I have packaged it into a file distribution system
Compile the program to ensure that the directory structure is as shown in the figure:
![Snipaste_2024-08-30_13-56-02](https://github.com/user-attachments/assets/bd8686ec-7904-4b7b-8c3f-fe814512da65)

### demo
**https://vip.123pan.cn/1818691018/url/demo.zip**

### Minimal reproduction project (MRP)

The project packaging is as follows:
**https://vip.123pan.cn/1818691018/url/scs.zip**

**Comment by E72UJ** [comment]
My resources are stored on the server in Chinese Mainland. If you have download problems, please contact me at any time

**Comment by akien-mga** [comment]
The project is too heavy for contributors to debug. To make a proper *minimal* reproduction project, you should try to remove content from the project and confirm that the bug still happens, until you get to the minimal set of scenes/code that trigger the issue.

In your case though, you're having a Vulkan GPU failure, which might be specific to your hardware / OS / drivers and difficult to reproduce for contributors. So narrowing it down to a specific project setup might help rendering contributor guess what feature may be badly supported on your hardware and how to work it around.

**Comment by akien-mga** [comment]
Could you check which version of the Nvidia drivers you have, and whether you can update them? That might fix the bug.

**Comment by E72UJ** [comment]
> 您能否检查一下您拥有的 Nvidia 驱动程序版本，以及是否可以更新它们？这可能会修复错误。

Thank you very much! It has been solved, I executed the Game Ready 560.94 graphics card driver update and the program has been successfully executed!

**Comment by E72UJ** [comment]
I executed the Game Ready 560.94 graphics card driver update and the program has been successfully executed!
